src/styles/main.scss

```
@@ -1,3 +1,9 @@
+@import "./utils/variables";
+@import "./utils/function";
+@import "./utils/mixins";
+
+@import "./blocks/calendar.scss";
+
 body {
   margin: 0;
```

```suggestion
  margin: 0;
  display: flex;
  align-items: center;
  justify-content: center;
  height: 100vh;
```
this styles make calendar in the center